cypress/integration/userInfo/UserInfo-test.spec.js

```diff
@@ -1,27 +1,24 @@
 // Copyright (c) Cosmo Tech.
 // Licensed under the MIT license.
 
-import { AppBarMenu } from '../../commons/actions/generic/AppBarMenu';
-import { PAGE_NAME } from '../../commons/constants/generic/TestConstants';
+import { AppBar } from '../../commons/actions';
 
 describe('UserInfo features', () => {
-  it('can log in with Microsoft account account', () => {
-    cy.visit(PAGE_NAME.SCENARIO);
-    cy.login();
-  });
-
   it('can change the application language', () => {
+    cy.login();
```

Comment:
Even if this is a single test here, could be interesting to put the code snippet 
```
before(() => {
    cy.login();
  });

  beforeEach(() => {
    cy.relogin();
  });
```
to uniformize tests?